pkg/server/wireexts_oss.go

```diff
@@ -81,6 +82,8 @@ var wireExtsBasicSet = wire.NewSet(
 	ossaccesscontrol.ProvideDatasourcePermissionsService,
 	wire.Bind(new(accesscontrol.DatasourcePermissionsService), new(*ossaccesscontrol.DatasourcePermissionsService)),
 	pluginsintegration.WireExtensionSet,
+	querycaching.ProvideService,
```

Comment:
are we talking about datasource caching or query caching here?

Reply:
This is talking about query caching, specifically the cache config per datasource